search-books/src/main/java/nix/education/entity/Tag.java

```diff
@@ -0,0 +1,26 @@
+package nix.education.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "tags")
+@Data
+@NoArgsConstructor
+public class Tag {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String tag;
+
+    public Tag(String tag) {
```

Comment:
Can we get rid of this constructor?

Reply:
Yes, we can, but it's the easiest way to create new Tag.